### PR TITLE
Fixing the align of commit stats in commit_page template.

### PR DESCRIPTION
**Pull request by puni9869**
Fixing the align center to row and space around for commit_page template.

screenshots 
before
image
image

after
image
image

### CODE REVIEW

File: templates/repo/diff/box.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div>
-	<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-ac gt-fw">
+	<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-fr gt-ac">
```
Comment (wxiaoguang) [inline templates/repo/diff/box.tmpl]: I do not think it's right to have `flex-direction: column;` on `diff-detail-box` but now use another `!important` to override it.

These `gt-` helpers are only for some simple/inline/fine-tune purpose, they shouldn't be mixed with other explicit styles.

Comment (puni9869) [reply templates/repo/diff/box.tmpl]: Yes, we can avoid `gt-fr`

Comment (puni9869) [reply templates/repo/diff/box.tmpl]: Fixed

**Review by wxiaoguang — APPROVED** [review]
I haven't tested but I guess I can accept either. 

ps: is the "gt-mt-0" still needed? If there is no margin on other classes, then maybe it could be removed.

silverwind has worked on this UI, mayby they could have some more thoughts.

**Comment by puni9869** [comment]
> 
image
we have this css rule in helper.css
Yes,  thats needed if you see the difference, in the gaps, its very minor, un-noticable.
image

**Review by lunny — APPROVED** [review]

**Pull request merged**